umbral/pre.py

```diff
@@ -108,6 +115,83 @@ def from_bytes(cls, capsule_bytes: bytes, curve: ec.EllipticCurve = None):
         components = splitter(capsule_bytes)
         return cls(*components)
 
+    def get_or_set_delegating_key(self, delegating_key: UmbralPublicKey = None):
```

Comment:
When I had the idea for `get_or_set`, I meant it to work for the model I proposed. I don't really like it here, tbh.

Reply:
OK.  I'll rework it then.